tsconfig.json

```diff
@@ -6,7 +6,7 @@
         "noUnusedLocals": true,
         "rootDir": "src",
         "strict": true,
-        "strictPropertyInitialization": false,
+        "strictPropertyInitialization": true,
```

Comment:
Just realised I changed this to see if it was raising errors when this is set to true. It might break other packages...

Reply:
Better to turn it back off for now, I believe. But yes, I'd love to turn this back on later - @tgalopin turned it off originally in sha: b3b836ba - but probably we will need to fix some other things when we do (any TypeScript fixing for any packages would be WARMLY welcomed, like this one) :)

Reply:
Ok, I gonna turn this off later today